apps/sms/views/shared/js/bootstrap.js

```diff
@@ -1,5 +1,18 @@
-/* global Startup */
+/* global App,
+          Startup */
 
+(function() {
 'use strict';
 
+var shimHostIframe = document.querySelector('.shim-host');
```

Comment:
nit: How about we move ConversationClient.init(App.instanceId) to here as well? Maybe it's better to aggregate the client init/service bootstrap in same place and startup doesn't need to reference App or shim related client. (I was thinking that this part could be in Startup.init but I'm fine if you want to sync with shim bootstrap)